system/Debug/Toolbar.php

```diff
@@ -468,4 +468,27 @@ protected function format(string $data, string $format = 'html'): string
 
 		return $output;
 	}
+
+	/**
+	 * Process the varable to string for display
+	 *
+	 * @param mixed $var Variable
+	 *
+	 * @return boolean|string
+	 */
+	protected function processVar($var)
```

Comment:
Let's make this private for now, unless there are compelling reasons to allow access to this method.